wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -761,7 +751,15 @@ extension ZMUserSession: ZMSyncStateDelegate {
         }
 
         mlsService.commitPendingProposalsIfNeeded()
-        fetchFeatureConfigs()
```

Comment:
I'm a bit uncomfortable about moving fetching the feature configs into the use case. We likely rely on the feature configs to be updated after each quick sync. But if it's in the use case we might assume it's only needed for that specific use case, which isn't true. If in the future we remove the use case, we'll unknowingly stop fetching feature configs and it would cause issues with features that rely on the configs to be up to date

Reply:
@johnxnguyen Thoughts

Reply:
yeah I also expressed this concern in our discussions and I agree, we should be more transparent. It would be better pull it out of the use case and invoke our new use case after it.

Reply:
But I think then we should add a comment stating that we should have fresh feature configs before trying to resolve 1-1s

Reply:
Can I create a ticket and handle it on a separate PR?